packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -221,11 +230,12 @@ export default class QuanticGeneratedAnswer extends LightningElement {
   isMaximumHeightExceeded() {
     // If we are still streaming add a little extra height to the answer element to account for the next answer chunk.
     // This helps a lot with the jankyness of the answer fading out when the chunk is close but not yet over the max height.
+    const maximumAnswerHeight = this.validateMaxCollapsedHeight();
```

Comment:
Following my understanding, `isMaximumHeightExceeded ` could be executed multiple times after each new generated answer which means that the `validateMaxCollapsedHeight` will also be executed multiple times which can yield multiple warning messages in the case where the new property contains an invalid value, which is not ideal.

The validation should be executed on first render imo and only once to avoid displaying the same warning multiple times.

Reply:
added a check for initial render so that it logs only once, thanks!